doc/changelog/03-notations/18089-master+fix18914-too-restrictive-notation-overlap-check.rst

```diff
@@ -0,0 +1,6 @@
+- **Fixed:**
```

Comment:
please put this directly in changes.rst
also a PR on master moving the changelog to the 8.19.2 section would be good